### ISSUE TITLE
Pre builded executables

### ISSUE THREAD

**Issue by Scientific-Guy** [dictu-lang/Dictu#404]
I wanted to use dictu but i see there is no executables to install rather i should manually build it from the source. It will be good if you make pre builded executables for people who do not want to build but want to use the language.

**Comment by Jason2605** [comment]
Definitely a good point, and something to look into. I wonder if there's a way to automate the release process a bit more which handles building Dictu on a range of different platforms.

**Comment by Scientific-Guy** [comment]
Yeah. Because only people who have cmake can only install and use this language and its a bad idea for those who don't have it. You can still keep custom building from the source code as a option but keeping the binary files in the releases tab will be better i guess.

**Comment by Jason2605** [comment]
While not strictly what you asked for (will still look into pre-built executables for releases), there is now an option to pull Dictu from [Dockerhub](https://hub.docker.com/r/dictulang/dictu) which may be of interest to you!
